src/py42/exceptions.py

```diff
@@ -263,6 +263,22 @@ def __init__(self, exception):
         super(Py42UpdateClosedCaseError, self).__init__(exception, msg)
 
 
+class Py42CaseNameExistsError(Py42BadRequestError):
+    """An error raised when trying to create a case with a name that already exists."""
+
+    def __init__(self, exception, case_name):
+        msg = "Case name '{}' already exists, please set another name".format(case_name)
+        super(Py42CaseNameExistsError, self).__init__(exception, msg)
+
+
+class Py42DescriptionLimitExceededError(Py42BadRequestError):
+    """An error raised when description of a case exceeds allowed char length limit."""
```

Comment:
when _the_ description of a case... (missing the)

Reply:
Done.